rustls-cert-gen/src/cert/mod.rs

```diff
@@ -0,0 +1,44 @@
+use rcgen::{Certificate, CertificateParams, Error};
+use std::{fs::File, io, path::Path};
+
+mod params;
+pub use params::BuildParams;
+
+mod ca;
+mod entity;
+mod signature;
+
+pub trait CertParams {
+	fn params(&self) -> &CertificateParams;
+}
+
+#[derive(Debug, Clone)]
+/// Struct to represent a Serialized Cert/Key pair
+pub struct SerializedEntity {
+	pub cert_pem: String,
+	pub key_pem: String,
```

Comment:
nit: I think we should emphasize this is the private key portion of a key _pair_ to distinguish from a public key.

Reply:
so just `private_key_pem`?